Source/Core/PolylineVolumeGeometry.js

```diff
@@ -163,7 +163,13 @@ define([
         }
 
         if (vertexFormat.tangent || vertexFormat.binormal) {
-            geometry = GeometryPipeline.computeBinormalAndTangent(geometry);
+            try {
+                geometry = GeometryPipeline.computeBinormalAndTangent(geometry);
+            } catch (e) {
+                console.log('Unable to compute tangents and binormals for polyline volume geometry');
```

Comment:
Ah, OK.  Leave it, but maybe remove the `console.log` or use something like `deprecationWarning` to avoid flooding the console.

Also use a debug pragma.